specs-go/config.go

```diff
@@ -90,8 +98,10 @@ type Mount struct {
 	// Source specifies the source path of the mount.  In the case of bind mounts on
 	// Linux based systems this would be the file on the host.
 	Source string `json:"source"`
-	// Options are fstab style mount options.
-	Options []string `json:"options,omitempty"`
+	// Options are fstab style mount options. (this field is platform dependent)
+	Options []string `json:"options,omitempty" platform:"linux,solaris"`
```

Comment:
On Wed, Jun 22, 2016 at 09:29:38AM -0700, John Howard wrote:
> > -	// Options are fstab style mount options.
> > -	Options []string `json:"options,omitempty"`
> > +	// Options are fstab style mount options. (this field is platform dependent)
> > +	Options []string `json:"options,omitempty" platform:"linux,solaris"`
> 
> I'm not sure - that PR is documentation.

The pattern so far in this repo has been to try and land the
(canonical, #342) Markdown docs in the same PR that updates the Go
types.  I think the change you're making to Mount.Options here makes
sense, and think that #439 has the associated Markdown updates.  Why
not make both of those changes in one atomic pivot PR?